src/Microsoft.AspNet.Identity/RoleManager.cs

```diff
@@ -60,6 +66,11 @@ public class RoleManager<TRole> : IDisposable where TRole : class
         protected IRoleStore<TRole> Store { get; private set; }
 
         /// <summary>
+        ///     Used to create scopes and messages for logging
+        /// </summary>
+        protected internal virtual IdentityLogger IdentityLogger { get; set; }
```

Comment:
Agree with @HaoK the property here can be just called Logger. the Logger in IdentityLogger is protected so no one can do Logger.Logger, right?